packages/execution-environments/src/common/BaseSnapExecutor.ts

```diff
@@ -11,16 +11,20 @@ import {
   JsonRpcRequest,
 } from '../__GENERATED__/openrpc';
 import { isJsonRpcRequest } from '../__GENERATED__/openrpc.guard';
-import { rpcMethods, RpcMethodsMapping } from './rpcMethods';
-import { sortParamKeys } from './sortParams';
 import { createEndowments } from './endowments';
 import { rootRealmGlobal } from './globalObject';
+import { rpcMethods, RpcMethodsMapping } from './rpcMethods';
+import { sortParamKeys } from './sortParams';
 
 type SnapRpcHandler = (
   origin: string,
   request: JsonRpcRequest,
 ) => Promise<unknown>;
```

Comment:
Can this be moved to the types package, to make it consumable for snaps?

Reply:
With difficulty - we would have to move `openrpc.json` as well and do generation of those types inside the types package. Don't think this should be done as a part of this PR.